### PR TITLE
Do not duplicate `CSS` variables declaration in `:root` selector (if so make a warning).

### PR DESCRIPTION
**Pull request by pszczesniak**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Type: Message. Closes #000.

---

### Additional information

*For example – encountered issues, assumptions you had to make, other affected tickets, etc.*

### CODE REVIEW

File: packages/ckeditor5-dev-build-tools/tests/plugins/splitCss/fixtures/duplicated-definitions/file2.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+:root {
+	--variable1: blue;
```
Comment (filipsobol) [inline packages/ckeditor5-dev-build-tools/tests/plugins/splitCss/fixtures/duplicated-definitions/file2.css:2]: I'd be good if this had different value from the `--variable1` in the `file1.css`. This way we could check if last definition of a given variable is used (the same as in CSS).

File: packages/ckeditor5-dev-build-tools/src/plugins/splitCss.ts [hunk 2]
```diff
@@ -119,11 +120,21 @@ function getDividedStyleSheetsDependingOnItsPurpose( rules: Array<Rule> ) {
 		editingViewStylesContent += objectWithDividedStyles.editingViewStyles;
 
 		if ( objectWithDividedStyles.rootDefinitions.length ) {
-			rootDefinitionsList.push( ...objectWithDividedStyles.rootDefinitions );
+			objectWithDividedStyles.rootDefinitions.forEach( definitionText => {
+				const listOfDefinitions = definitionText.split( '\n' );
+
+				listOfDefinitions.forEach( definition => {
+					if ( definition && rootDefinitionsSet.has( definition ) ) {
+						showInfoOfDuplicatedVariableDefinition( definition );
```
Comment (filipsobol) [inline packages/ckeditor5-dev-build-tools/src/plugins/splitCss.ts:128]: Variable defined later should override previous value (the same as in CSS).

**Review by filipsobol — CHANGES_REQUESTED** [review]

File: packages/ckeditor5-dev-build-tools/src/plugins/splitCss.ts (hunk 2 above)
Comment (pszczesniak) [reply packages/ckeditor5-dev-build-tools/src/plugins/splitCss.ts:128]: TBH whole method should be modified. I choose `Set` to quickly eliminate the duplicates, but each `rootDefinitionsSet` helds a string like `--variable: red;` and it compares it with same one not just a variable name.

To make it work as expected `Set` should be changed into `Map` and store values as key/value: `'--variable'` => `'red'`. Check if the key already exists and if yes, replace just the value (and can make some warning that value `X` of variable `FOO` was changed to `Y` ).

**Comment by pszczesniak** [comment]
Decision was made to not do it now and wait when we wants to add/modified existing source maps to splitted `CSS` files.

**Pull request closed without merging**